sdk/src/wallet/core/operations/address_generation.rs

```diff
@@ -18,12 +16,7 @@ impl Wallet {
     /// Generate an address without storing it
     /// ```ignore
     /// let public_addresses = wallet
-    ///     .generate_ed25519_addresses(
-    ///         0,
-    ///         false,
-    ///         0,
-    ///         None,
-    ///     )
+    ///     .generate_ed25519_address(None)
     ///     .await?;
     /// ```
     pub async fn generate_ed25519_address(
```

Comment:
I think we should remove this from the wallet, users can do it with the secret manager

Reply:
that's a significant change though, that has a lot of downstream effects. I see that it's related to turning the wallet into a single account one, but not that strongly that it couldn't be a separate PR, wdyt?

Reply:
Fine, opened an issue https://github.com/iotaledger/iota-sdk/issues/1536